draft.md

```diff
@@ -25,21 +25,20 @@ helper functions for common cleaning tasks. Everything you need to know about
 how to use it is on the [package website](https://ropensci.github.io/jstor/).
 You will find three vignettes, with a general introduction, examples on how to
 import many files at once, and a few examples of known quirks of JSTORs data.
-There is also a lengthy case study that also shows how to combine metadata and
-data about the content of the articles. But for now let us turn back the clock
+There is also a lengthy case study that shows how to combine metadata and
+content of the articles. But for now let us turn back the clock
 to follow along my journey of developing the package.
 
 # Hacking Away
 Back in March 2017, I was starting out as a MA-student of
 sociology in a research project
 concerned with the scientific elite within sociology and economics. The project
-had many goals, but writing an R package was not one of them. At the beginning
-of my engagement,
+had many goals, but writing an R package was not one of them. At the start,
 I was presented with a dataset which was huge, at least for my terms: 
 around 30GB of data, half of which was text, the other half 500,000 `.xml`-files.
 The dataset was incredible in its depth: we basically sat on all articles
 from JSTOR which belonged to the topics "sociology" and "economics". To repeat:
-all articles that JSTOR has on those topics for all years.
+all articles that JSTOR has on those topics for all the time JSTOR has data on.
 
 My task was to somehow make this data accessible for our research. Since we are
 sociologists and no computer experts and my knowledge of R was mainly 
```

Comment:
"but there would probably be more efficient solutions" klingt hier so als hättest du so damit angefangen, und dann doch noch was besseres gefunden, was jedoch nicht der Fall ist. Das ist irreführend, man wartet dann die ganze Zeit darauf dass du ein besseres Konzept bringst ;)